dev/integration_tests/ios_add2app/build_and_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")"
+
+pushd flutterapp
+../../../../bin/flutter build ios --debug --no-codesign -v
```

Comment:
is -v necessary? Might make logs harder to parse.

Reply:
It's how we do almost all our tests...  we could miss some output on a failure if we don't do a -v (e.g. stacktraces aren't printed)